src/neural-net-visualizer/visualizer/js/visualizer.js

```diff
@@ -1,3 +1,9 @@
+
+console.log = function(...args) {
+  const message = util.format(...args);
```

Comment:
`util` does not typically exist in a front-end environment, will need to be replaced with something else